src/Microsoft.ML.Data/MLContext.cs

```diff
@@ -79,6 +79,13 @@ public sealed class MLContext : ISeededEnvironment
         /// </summary>
         public ComponentCatalog ComponentCatalog => _env.ComponentCatalog;
 
+        private static string _tempFilePath = System.IO.Path.GetTempPath();
+        public static string TempFilePath
```

Comment:
This shouldn't be `static`. It should be an instance property on MLContext.

Reply:
The reason I made it static is there are places we don't pass the ML Context to that use a temp path. Do you have some time to sync on this today?